api/server/httputils/errors.go

```diff
@@ -54,6 +54,7 @@ func GetHTTPErrorStatusCode(err error) int {
 			code    int
 		}{
 			{"not found", http.StatusNotFound},
+			{"cannot find", http.StatusNotFound},
```

Comment:
Just to be clear. You want to return NewRequestNotFoundError from platform specific wrapper to execv/CreateProcess?